src/perform_step/rkc_perform_step.jl

```diff
@@ -16,51 +16,44 @@ end
   # The number of stage.
   mdeg = Int(floor(sqrt((1.5 + dt * integrator.eigen_est)/0.811) + 1))
   if mdeg >= 200
-    h = 0.8 * (200 ^ 2 * 0.811 - 1.5)/integrator.eigen_est
+    dt = 0.8 * (200 ^ 2 * 0.811 - 1.5)/integrator.eigen_est
```

Comment:
this won't actually update the `dt`. A more sophisticated approach will be required here. Is the `dt` only ever decreased by this?